cg/meta/demultiplex/demux_post_processing.py

```diff
@@ -126,10 +124,116 @@ def finish_flow_cell_temp(self, flow_cell_name: str) -> None:
         self.status_db.session.commit()
 
         # 5. Store flow cell data in housekeeper.
+        self.add_flow_cell_data_to_housekeeper(
+            flow_cell=flow_cell, flow_cell_directory=flow_cell_dir
+        )
 
         # 6. Create sequencing metrics
         self.add_sample_lane_sequencing_metrics_for_flow_cell(flow_cell_name=flow_cell_name)
 
+    def add_flow_cell_data_to_housekeeper(
+        self, flow_cell: Flowcell, flow_cell_directory: Path
+    ) -> None:
+        """Add flow cell data to housekeeper."""
+        LOG.info(f"Add flow cell data to housekeeper for {flow_cell.name}")
+
+        self.add_bundle_and_version_if_not_exists(flow_cell_name=flow_cell.name)
+
+        tags: List[str] = [SequencingFileTag.FASTQ, SequencingFileTag.SAMPLE_SHEET, flow_cell.name]
```

Comment:
It seems unnecessary to always try to add these constant tags (FASTQ and SAMPLE_SHEET).

Reply:
I guess the only time this would fail for us is if we start with a completely empty database - if they were to be removed. Having them should make it fool-proof and I guess this check does not add much cognitive load nor time running the process. I would leave it as it is.